.github/workflows/linux-build-and-test.yaml

```diff
@@ -281,8 +273,9 @@ jobs:
 
     - name: Upload test results to the database
       # Don't upload the results of the flaky check, because the db schema only
-      # supports running one test once per job.
-      if: always() && (! contains(matrix.name, 'Flaky'))
+      # supports running one test once per job. Also disable uploading of the
```

Comment:
Could you add this comment to `ci_settings.py` as well?  I guess otherwise, we will forget to remove this condition after changing the PG version.

Reply:
Why do we have to disable it? It only needs psql, we can use the one we just built.

Reply:
@jnidzwetzki added a TODO comment in that file